editor/src/clj/editor/console.clj

```diff
@@ -265,7 +273,18 @@
                             :style-class "console-filter-popup-background"}
                            {:fx/type fx.v-box/lifecycle
                             :children
-                            [{:fx/type fx.list-view/lifecycle
+                            [{:fx/type fx.check-box/lifecycle
+                              :focus-traversable false
+                              :max-width ##Inf
+                              :v-box/margin 4
+                              :id "global-console-filtering"
+                              :selected (prefs/get prefs console-filtering-key)
+                              :on-action {:event-type :toggle-global-filtering}
+                              :on-selected-changed (partial set-global-filtering prefs)
```

Comment:
I would use `:on-selected-changed {:event-type :toggle-global-filtering}` instead, and I'd updated the prefs in the `handle-filter-event!` function like we do for filter changes. It would be more consistent that way.